src/System.CommandLine.Subsystems/Directives/DiagramSubsystem.cs

```diff
@@ -35,7 +35,7 @@ internal static StringBuilder Diagram(ParseResult parseResult)
         var builder = new StringBuilder(100);
 
 
-        Diagram(builder, parseResult.RootCommandResult, parseResult);
+        //Diagram(builder, parseResult.RootCommandResult, parseResult);
```

Comment:
Discussion from meeting clarified this is being commented out due to it being broken.  Can we get a todo on fixing this to capture this?